common/horizon/etc/horizon.env

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 
 export DATABASE_URL="postgres://stellar:__PGPASS__@localhost/horizon"
-export STELLAR_CORE_URL="http://localhost:11626"
+export STELLAR_CORE_URL="http://localhost:11726"
 export STELLAR_CORE_BINARY_PATH=/usr/bin/stellar-core
 export LOG_LEVEL="info"
 export ENABLE_CAPTIVE_CORE_INGESTION="true"
 export CAPTIVE_CORE_USE_DB=true
-export STELLAR_CAPTIVE_CORE_HTTP_PORT=0
+export STELLAR_CAPTIVE_CORE_HTTP_PORT=11726
```

Comment:
@stellar/platform-committers Something I've noticed when switching to using the captive core for the stellar-core, is that horizon takes a lot longer before it starts reporting stats about the stellar-core on its home URL.

When I use the external stellar-core, stats for the cores available protocol version, and which ledger it is up to, display almost immediately.

When I use the internal captive core, those same stats don't show up until the stellar-core is fully caught up and ingesting live data.

What's the reason for this change in behavior?

This change in behavior is why I have marked the horizon tests in this PR to not expect those values to be present in the response when testing pubnet. Because it takes too long for a full catch up on pubnet.

Reply:
@leighmcculloch this is probably due to https://github.com/stellar/go/blob/master/services/horizon/internal/app.go#L216-L222

If horizon is ingesting from an empty db horizon will first need to populate its db with all the ledger entries from the history archives. On pubnet, this operation takes at least 1 hour. During that period horizon is not ingesting from captive core and so the captive core instance remains blocked until horizon reads tx meta from the captive core pipe. While captive core is blocked it doesn't respond to any HTTP requests.